clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/KubernetesUtil.groovy

```diff
@@ -112,4 +85,29 @@ class KubernetesUtil {
     return securityGroups
   }
 
+  static String securityGroupKey(String securityGroup) {
+    return String.format("$SECURITY_GROUP_LABEL_PREFIX%s", securityGroup)
```

Comment:
While I don't put it everywhere, I prefer to include the `return` statement. Wouldn't bother me much to see it left out in a one-line method though.

Fitting in with the neighbors should be the rule of thumb when unsure.